numba/cuda/tests/cudapy/test_ufuncs.py

```diff
@@ -154,6 +173,21 @@ def test_degrees_ufunc(self):
     def test_radians_ufunc(self):
         self.basic_ufunc_test(np.radians, kinds='f')
 
+    def test_bitwise_and_ufunc(self):
+        self.basic_int_ufunc_test(np.bitwise_and)
+
+    def test_bitwise_or_ufunc(self):
+        self.basic_int_ufunc_test(np.bitwise_or)
+
+    def test_bitwise_xor_ufunc(self):
+        self.basic_int_ufunc_test(np.bitwise_xor)
+
+    def test_invert_ufunc(self):
+        self.basic_int_ufunc_test(np.invert)
+
+    def test_bitwise_not_ufunc(self):
+        self.basic_int_ufunc_test(np.bitwise_not)
+
```

Comment:
Could you add a comment here explaining why there's no test for left and right shift? I don't think it's necessary to duplicate the entire comment from `numba/tests/test_ufuncs.py`, just referring to it would be fine.